autoextract_poet/inputs.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+from typing import Any, Dict, Optional
+
+
+@dataclass
+class ResponseData:
+    """Represent an AutoExtract API response containing returned data."""
+    data: Optional[Dict[str, Any]]
+
+
+@dataclass
+class ProductResponseData(ResponseData):
+    """Represent an AutoExtract Product response."""
```

Comment:
```suggestion
    """ AutoExtract Product response."""
```

and the same for other data types :)

Reply:
Done.